src/client/assets/images/index.ts

```diff
@@ -2,6 +2,5 @@ import AlchemyCertified from './alchemy-certified.svg';
 import YearnLogoFull from './yearn-logo-full.svg';
 import YearnLogo from './yearn-logo.svg';
 import MetamaskLogo from './metamask-logo.png';
-import AmsterdamLogo from './amsterdam-banner.jpg';
 
-export { AlchemyCertified, YearnLogoFull, YearnLogo, MetamaskLogo, AmsterdamLogo };
```

Comment:
For testing purpose, I used the AmsterdamBanner as example of integration. Because of that, we no longer need that